### ISSUE TITLE
Fails to build through Carthage on Xcode 7.3

### ISSUE THREAD

**Issue by almassapargali** [watson-developer-cloud/ios-sdk#222]
I get following output:
```
$ xcodebuild -version
Xcode 7.3
Build version 7D175
$ carthage update --platform iOS --no-use-binaries
...
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:137:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:123:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextAudioStreamer.swift:186:15: error: ambiguous use of 'writeData'
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:123:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:137:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:151:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:137:19: error: method does not override any method from its superclass
/Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift:123:19: error: method does not override any method from its superclass
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal arm64 /Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextAudioStreamer.swift
	CompileSwift normal arm64 /Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToTextWebSocket.swift
	CompileSwift normal arm64 /Users/almassapargali/code/speaker-dev/Carthage/Checkouts/ios-sdk/WatsonDeveloperCloud/SpeechToText/SpeechToText.swift
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
(4 failures)
```

**Comment by glennrfisher** [comment]
Hmm. Interesting. Let me check if I see the same behavior on my machine.

**Comment by almassapargali** [comment]
Btw, not exactly sure that subclassing WebSocket is good idea, I don't think it'd designed for subclassing. Maybe instead these classes should be wrapper around WebSocket with it's own API?

**Comment by almassapargali** [comment]
Issue is this library missing `completion: (() -> ())? = default` parameters when overriding `Starscream` functions.

**Comment by glennrfisher** [comment]
Looks like the completion handlers were added with daltoniam/Starscream/pull/178, which was released an hour ago with [Starscream v1.1.3](https://github.com/daltoniam/Starscream/releases/tag/1.1.3).

**Comment by almassapargali** [comment]
Ok, then I guess `carthage bootstrap --platform iOS --no-use-binaries` should help, since this project's `Cartfile.resolved` lists `github "daltoniam/Starscream" "1.1.2"`.

Thanks for investigating! You can close this issue, or keep it to track this change.

**Comment by glennrfisher** [comment]
Great point! This is a great example of the use of `bootstrap` instead of `update`.

**Comment by glennrfisher** [comment]
Fixed with PR #223 and [Release 0.2.4](https://github.com/watson-developer-cloud/ios-sdk/releases/tag/v0.2.4). Thanks for raising this issue, @almassapargali!
